webpack.common.js

```diff
@@ -1,10 +1,13 @@
 const path = require('path');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
+require('babel-core/register');
```

Comment:
You don't need to include these modules. Moreover you don't need babel-register at all